docs/en/README.md

```diff
@@ -63,15 +63,15 @@ Demo Environment:
     * Apollo also provides a packaging script, which can generate all required installation packages with just one click, and supports customization of runtime parameters
 
 # Usage
-1. [Apollo User Guide](en/usage/apollo-user-guide)
-2. [Java SDK User Guide](en/usage/java-sdk-user-guide)
-3. [.Net SDK user Guide](en/usage/dotnet-sdk-user-guide)
-4. [Third Party SDK User Guide](en/usage/third-party-sdks-user-guide)
-5. [Other Language Client User Guide](en/usage/other-language-client-user-guide)
-6. [Apollo Open APIs](en/usage/apollo-open-api-platform)
+1. [Apollo User Guide](en/portal/apollo-user-guide)
+2. [Java SDK User Guide](en/client/java-sdk-user-guide)
+3. [.Net SDK user Guide](en/client/dotnet-sdk-user-guide)
+4. [Third Party SDK User Guide](en/client/third-party-sdks-user-guide)
```

Comment:
This link currently points to the old `/en/client/third-party-sdks-user-guide`. However, it has been divided into more specific guides like `golang-sdks-user-guide`, `nodejs-sdks-user-guide`, etc. Should we continue to keep both versions moving forward?

Reply:
No, I think it should be removed.